woocommerce/woocommerce-android
# Dashboard empty view with stats

Fixes #994.

To test:
1. Connect to a store that has no orders
2. Go to the Dashboard tab
3. Notice the empty view is displayed with the daily visitors stats
4. Switch to the Orders tab
5. Notice the empty view does not have any stats
6. Connect to a store with orders
7. Go to the Dashboard tab
8. Verify the visitors, orders and revenue stats are loaded as expected

 
![image](https://user-images.githubusercontent.com/1522856/60576167-2fcdcc80-9d7d-11e9-9851-72835ec60290.png)

Update release notes:
